website/docs/getting-started/setup/installation-guides/aws-ami.md

```diff
@@ -1,145 +1,113 @@
 ---
-description: Deploy Appsmith on AWS with an AMI on the marketplace
+description: Deploy Appsmith on AWS with an AMI available on the AWS marketplace.
```

Comment:
The AMI always pulls the latest Appsmith image, and users sometimes get confused with the AMI release version and the Appsmith release, since we don't update it very often.
I think we should also mention that. We will just use the major version of Appsmith that maps to the AMI version.
In this case v1.9